### ISSUE TITLE
Travis CI token does not work

### ISSUE THREAD

**Issue by isaachier** [opentracing/opentracing-c#7]
Running into the same issue as described in this SO question: https://stackoverflow.com/q/31159275/1930331. Probably need someone with access to opentracing org to generate a token for this use.

**Comment by yurishkuro** [comment]
Travis was disabled for this repo, I just re-enabled it.

https://travis-ci.org/opentracing/opentracing-c/builds

PS - please add the build badge to the README

**Comment by isaachier** [comment]
Good idea, but still an issue: https://travis-ci.org/opentracing/opentracing-c/builds/361849379.

**Comment by yurishkuro** [comment]
It's not clear to me what's failing. The main build (in the first matrix row) exited with
```
All tests compiled and passed
The command "./scripts/build.sh "$HOME/install"" exited with 0.
```

The two after_install steps also seems successful. Only the dpl (deploy?) step is failing. Where is this coming from: `invalid option "--github-token="`?

**Comment by isaachier** [comment]
I believe it is the deploy stage. This is where it uploads the generated documentation to gh-pages.

**Comment by yurishkuro** [comment]
That sounds like a fairly routine thing to do, i.e. made a commit from Travis run. Did you search for the docs for that?

Is that deploy going to run on every PR or only from master?

**Comment by isaachier** [comment]
Originally only from master but tweaked it when I swapped the token earlier today. I did look into it briefly and found an almost exact match on StackOverflow. It should be linked in the original issue above.

**Comment by yurishkuro** [comment]
I think Deploy Keys are the answer: https://developer.github.com/v3/guides/managing-deploy-keys/

**Comment by isaachier** [comment]
OK I still think that requires some assistance from one of the OT admins. Let me know if I can help out. Thanks.

**Comment by yurishkuro** [comment]
I'd love to help but I don't know what exactly you need. I just made you an admin of this repo, this should be sufficient to do anything that I would be able to do, the only exception I know of is if a 3rd party site wants access to the repository then that access is granted at the org level, but I don't think you need that.

**Comment by isaachier** [comment]
Thanks! I really doubt I need that at the moment. Actually, submitted a PR for now to avoid doing the doc upload in Travis build. That keeps things simple for now and builds succeed. Can always upload docs manually.
